tooling/tsconfig/base.build.json

```diff
@@ -9,6 +9,8 @@
     "resolveJsonModule": true,
     "moduleDetection": "force",
     "isolatedModules": true,
+    "noErrorTruncation": true,
+    "exactOptionalPropertyTypes": true,
 
     /* Strictness */
```

Comment:
Aren't comments not valid in JSON?

Reply:
they are in jsonc, which tsconfigs are. but github doesn't recognize the `base.build.json` file as that so it doesn't interpret it as jsonc

Reply:
Ah 👍